georiviere/portal/serializers/portal.py

```diff
@@ -40,11 +40,12 @@ class Meta:
         )
 
     def get_flatpages(self, obj):
-        return FlatpagePortalSerializer(obj.flatpages, many=True, context={'portal_pk': self.context.get('portal_pk')}).data
+        return FlatpagePortalSerializer(obj.flatpages, many=True,
+                                        context={'portal_pk': self.context.get('portal_pk')}).data
 
     def to_representation(self, instance):
         ret = super().to_representation(instance)
-        layers_without_group = instance.layers.filter(group_layer__isnull=True)
+        layers_without_group = instance.available_layers.filter(group_layer__isnull=True)
         if layers_without_group.exists():
```

Comment:
```suggestion
        if bool(layers_without_group):
```
https://docs.djangoproject.com/fr/4.2/ref/models/querysets/#django.db.models.query.QuerySet.exists